### DISCUSSION TITLE
Commas get removed from wildcards - is this a bug???

### DISCUSSION THREAD

**Discussion by rtaskf** [adieyal/sd-dynamic-prompts#672] [General]
I think the latest update broke something... All my wildcard files are useless now.

When I write {|hot,}{|summer,}{|blue sky,} it doesn't turn into hot,summer,blue sky, anymore - it becomes hotsummerblue sky.

Can you please fix this? I can no longer use my extensive wildcard "library".

**Comment by adieyal** [comment]
@akx looks like it was broken in v0.30.0

**Reply by akx** [reply]
Alright, tracking this in https://github.com/adieyal/dynamicprompts/issues/106. Might not have the time to fix it today though. Sorry for the inconvenience!

**Comment by rtaskf** [comment]
Thanks a lot for dealing with it so quickly!

**Reply by akx** [reply]
This corner case is now fixed in dynamicprompts 0.30.4. I've set that version as a requirement in 0645c944d379c52e9f97bff7d24096f82165c59c, so if you update the extension, it should pull that in.
